distribution/errors.go

```diff
@@ -18,11 +18,11 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-// ErrNoSupport is an error type used for errors indicating that an operation
+// notSupportedError is an error type used for errors indicating that an operation
 // is not supported. It encapsulates a more specific error.
-type ErrNoSupport struct{ Err error }
+type notSupportedError struct{ Err error }
 
-func (e ErrNoSupport) Error() string {
+func (e notSupportedError) Error() string {
```

Comment:
I can't find any place where an `ErrNoSupport` value is constructed. It appears that this type can be deleted outright!

Reply:
Ah, nice catch; yes indeed